experiment.py

```diff
@@ -38,18 +38,19 @@ def print_fig2_results(cc_proto):
             lines = f.readlines()
             avg_capacity = float(lines[0].split(' ')[2])
             avg_throughput = float(lines[1].split(' ')[2])
-            queueing_delay = float(lines[2].split(' ')[5])
+            queuing_delay = float(lines[2].split(' ')[5])
```

Comment:
I think that it was spelled correctly before

Reply:
https://english.stackexchange.com/questions/60852/queueing-or-queuing?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
I guess whatever editor I was using said the latter was the correct spelling, but looks like they're both ok!  So let's just leave it like this

Reply:
lol